src/Docs/Resources/current/50-how-to/350-how-to-log.md

```diff
@@ -0,0 +1,64 @@
+[titleEn]: <>(How to log)
+[metaDescriptionEn]: <>(This article shows you how to configure default logger settings and create logger instances)
+[hash]: <>(article:how_to_log)
+
+## Overview
+
+Shopware 6 uses Monolog loggers and provide a central factory for the most common logging cases.
+
+## Setup a logger
+
+You can use the LoggerFactory class to generate a logger service within your service definition:
+
+<div class="tabs">
+    <nav>
+        <a href="#">XML</a>
+        <a href="#">YML</a>
```

Comment:
where is php :D

Reply:
Do you support symfony 5?

Reply:
You only load services.yml/.xml

Reply:
Sure it works
https://github.com/shopware/platform/blob/master/src/Core/Framework/Bundle.php#L146

Reply:
I had it different in mind but hey, I approve. Let's do it in php as well.

I had it in mind that there is a locator that looks for `services.{xy}ml`.